project/frontend/src/components/RestaurantCardStack.js

```diff
@@ -0,0 +1,89 @@
+import './RestaurantCardStack.css';
+
+import React, { useState } from 'react';
+
+import RestaurantCard from './RestaurantCard.js';
+
+/**
+ * A navigable stack of restaurant cards.
+ *
+ * @param {!Array<Object<string, *>>} props.cards A list of restaurant objects whose
+ * data will be displayed in RestaurantCard components within this stack.
+ */
+function RestaurantCardStack(props) {
+  const [currentCardIndex, setCurrentCardIndex] = useState(0);
+
+  const maxScaleFactor = 1.0;
+  const minScaleFactor = 0.85;
+  const numberOfCollapsedCards = 3;
+  const uncollapsedCardHeight = 487;
+  const offsetBetweenCollapsedCards = 24;
+  const offsetBeforeUncollapsedCard = 152;
```

Comment:
Wondering if you'd want to make these relative to each other so that if you have to resize, the proportions will stay constant? Not fully sure how the whole design works but just a thought

Reply:
I think this is a good idea, but I'd like to put it off until I implement everything around resizing the cards based on screen size because I'm not sure what that's going to look like yet and changes now  will probably just get overwritten when I actually do the whole refactor.